neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/embedded/impl/EmbeddedNeo4jGraphDatabaseFactory.java

```diff
@@ -50,6 +58,18 @@ public void initialize(Map<?, ?> properties) {
 
 	@Override
 	public GraphDatabaseService create() {
+		LOG.infof( "1. Try  create new service instance for dbPath  %s", dbLocation );
+		final String dbLocationAbsolutePath = dbLocation.getAbsolutePath();
+		LOG.infof( "2. Try  create new service instance for dbPath  %s", dbLocationAbsolutePath );
+		return GRAPH_DATABASE_SERVICE_MAP.computeIfAbsent( dbLocationAbsolutePath, ( String path ) -> {
+			LOG.infof( " Create new service instance for dbPath  %s", dbLocationAbsolutePath );
+			return buildGraphDatabaseService();
```

Comment:
Why do you need the same log 3 times? Can't you just print the absolutePath? I would prefer not to add a lot of redundant information in the output of the log,